### PR TITLE
Debug/fix crashing issue

### PR DESCRIPTION
**Pull request by cmanamela**
fixed problems that came with the bulk merging that happened on friday

### CODE REVIEW

File: waterlog/src/Globals.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export const Globals = {
-	API_URL: 'https://api.iot.retrotest.co.za',
+	API_URL: 'https://localhost:44383',
```
Comment (ash-rabbit) [inline waterlog/src/Globals.js]: is this a mistake or you want it to go into Dev?

File: waterlog/src/actions/TankLevelsActions.js [hunk 2]
```diff
@@ -20,12 +20,13 @@ import {
...
   
-  export const fetchTankLevels = () => dispatch => {
+  export const fetchTankLevels = () => dispatch => { 
     dispatch(fetchTankLevelsBegin());
-    fetch(`${Globals.API_URL}/api/Tanklevels`)
+    fetch(`${Globals.API_URL}/api/tanklevels`)
       .then(handleErrors)
       .then(res => res.json())
       .then(levels => {
+        console.log('levels', levels)
```
Comment (ash-rabbit) [inline waterlog/src/actions/TankLevelsActions.js]: Do you wanna test this in Dev or you forgot to remove the console.log('levels', levels)

File: waterlog/src/reducers/TankLevelReducer.js [hunk 3]
```diff
@@ -12,18 +12,21 @@ const initialState = {
...
         case FETCH_TANK_LEVELS_SUCCESS:
+            console.log('levels', action.payload)
+            console.log('levels', action.payload.levels)
             return {
-                ...state,
-                loading: false,
-                items: action.payload.levels,
-                error: null
-            }
+                    ...state,
```
Comment (laurent-mbuyu) [inline waterlog/src/reducers/TankLevelReducer.js:22]: Why are you returning the initial or default object with the modified object in your reducer?

File: waterlog/package.json [hunk 4]
```diff
@@ -23,7 +23,6 @@
     "react-chartjs-2": "^2.7.4",
     "react-dom": "^16.7.0",
     "react-leaflet": "^2.2.0",
-
```
Comment (ash-rabbit) [inline waterlog/package.json]: Why is this file being committed?

Comment (cmanamela) [reply waterlog/package.json]: I didn't know that it isn't supposed to be committed, shouldn't it be?

Comment (ash-rabbit) [reply waterlog/package.json]: No it shouldn't

File: waterlog/src/reducers/TankLevelReducer.js (hunk 3 above)
Comment (LisaQ04) [reply waterlog/src/reducers/TankLevelReducer.js:22]: I set the initial state first. I am not really sure why it is being done like this. But of all the reducer examples I saw. They use the same approach.

File: waterlog/src/actions/TankLevelsActions.js (hunk 2 above)
Comment (cmanamela) [reply waterlog/src/actions/TankLevelsActions.js]: I removed those comments

File: waterlog/src/Globals.js (hunk 1 above)
Comment (cmanamela) [reply waterlog/src/Globals.js]: fixed

File: waterlog/package.json (hunk 4 above)
Comment (cmanamela) [reply waterlog/package.json]: i didn't know that it is not supposed to be committed. Should i remove it?

File: waterlog/src/reducers/TankLevelReducer.js (hunk 3 above)
Comment (laurent-mbuyu) [reply waterlog/src/reducers/TankLevelReducer.js:22]: I'm pretty reluctant about this approach.
Knowing that the reducer returns the correct object based on the action type selected, I would suggest returning the default state when it fulfills its specific condition else return the new state.

Seeing it being done like that should be questioned when you don't understand it or don't approve of.

Comment (LisaQ04) [reply waterlog/src/reducers/TankLevelReducer.js:22]: I have been doing it the same way for more than one reducer and @clewarne has approved of how I was returning the object.

Comment (laurent-mbuyu) [reply waterlog/src/reducers/TankLevelReducer.js:22]: @clewarne Help me to understand why @LisaQ04 is returning the initial or default object with the modified object in his reducer?

Comment (clewarne) [reply waterlog/src/reducers/TankLevelReducer.js:22]: @laurent-mbuyu what is happening here is that we don't want to lose any of the initial properties on the state. Not all properties always change so we first make sure to keep everything on the state and then only overriding the changed properties
```{
   ...state,
   // here override any properties that need to change
```

**Review by ash-rabbit — APPROVED** [review]

File: waterlog/src/Components/TankComponent.js [hunk 5]
```diff
@@ -72,6 +72,7 @@ export class TankComponent extends Component {
```
Comment (grootendorste) [inline waterlog/src/Components/TankComponent.js:73]: Just a question, are you trying to add spaces between the heading tags? I'd rather add margins or padding to the style for this

Comment (grootendorste) [reply waterlog/src/Components/TankComponent.js:73]: ...margins or padding depending on the background aesthetics of the objects. It just keeps the markup a bit cleaner.

**Review by grootendorste — CHANGES_REQUESTED** [review]
Just some code stuff.

File: waterlog/src/Globals.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
 export const Globals = {
-	API_URL: 'https://api.iot.retrotest.co.za',
+	API_URL: 'https//api.iot.retrotest.co.za',
```
Comment (Tebogo-Seshibe-Retro) [inline waterlog/src/Globals.js]: Why did you remove the colon?

**Review by Tebogo-Seshibe-Retro — CHANGES_REQUESTED** [review]

**Review by Tebogo-Seshibe-Retro — APPROVED** [review]

**Review by clewarne — APPROVED** [review]

**Review by CeeyarRbt — APPROVED** [review]

File: waterlog/src/actions/TankLevelsActions.js [hunk 7]
```diff
@@ -20,9 +20,9 @@ import {
     payload: { error }
   });
   
-  export const fetchTankLevels = () => dispatch => {
+  export const fetchTankLevels = () => dispatch => { 
     dispatch(fetchTankLevelsBegin());
-    fetch(`${Globals.API_URL}/api/Tanklevels`)
+    fetch(`${Globals.API_URL}/api/tanklevels`)
```
Comment (CilliersN) [inline waterlog/src/actions/TankLevelsActions.js:25]: Note. when it was created, it should've been done with camel case
